AccountContacts/mex_definition/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+    "summary": "Manage related contact information",
+    "revisionCount": 1,
+    "contextObject": "Jobs",
+    "displayOrder": 0,
+    "hasAttachments": ["Contacts"],
```

Comment:
This Form doesn't use any attachments, so you should get rid of it

Reply:
thank you!